packaging/centos/datadog-agent.init

```diff
@@ -27,7 +27,7 @@ AGENTCONF="/etc/dd-agent/datadog.conf"
 DOGSTATSDPATH="/opt/datadog-agent/agent/dogstatsd.py"
 AGENTUSER="dd-agent"
 PIDPATH="/var/run/dd-agent/"
-FORWARDERPATH="/usr/bin/dd-forwarder"
+FORWARDERPATH="/opt/datadog-agent/agent/ddagent.py"
```

Comment:
They are already `755` in the repo, and are only [copied](https://github.com/DataDog/dd-agent-omnibus/blob/master/config/software/datadog-agent.rb#L30) during the build, and [`chown`](https://github.com/DataDog/dd-agent-omnibus/blob/master/package-scripts/datadog-agent/postinst#L54) during the postinstall. They should stay `a+x` unless the build machine has a weird `umask`.